content/v2.0/introduction/getting-started.md

```diff
@@ -138,29 +162,1243 @@ Visit your application in the browser at [http://localhost:2300](http://localhos
+
+Finished in 0.04572 seconds (files took 0.72148 seconds to load)
+1 example, 1 failure
+```
+
+To fix this, let's open our application's routes file at `config/routes.rb`:
+
+```ruby
+# config/routes.rb
+
+# frozen_string_literal: true
+
+module Bookshelf
+  class Routes < Hanami::Routes
+    root { "Hello from Hanami" }
+  end
+end
+```
+
+This `Bookshelf::Routes` class contains the configuration for our application's router. Routes in Hanami are comprised of a HTTP method, a path, and an endpoint to be invoked, which is usually a Hanami action. (See the [Routing guide](/v2.0/routing/overview/) for more information).
+
+We'll take a look at adding more routes in a moment, but for now let's get our spec to pass. The above `Bookshelf::Routes` class contains only one route, the `root` route, which handles `GET` requests for `"/"`.
+
+Rather than invoking an action, this route is configured to invoke a block, which returns "Hello from Hanami".
+
+Blocks are convenient, but let's adjust our route to invoke an action instead:
+
+```ruby
+# config/routes.rb
+
+# frozen_string_literal: true
+
+module Bookshelf
+  class Routes < Hanami::Routes
+    root to: "home.show"
+  end
+end
+```
+
+If we run our test again, we'll see a `Hanami::Routes::MissingActionError`:
+
+```shell
+Failures:
+
+  1) Root is successful
+     Failure/Error: get "/"
+
+     Hanami::Routes::MissingActionError:
+       Could not find action with key "actions.home.show" in Bookshelf::App
+
+       To fix this, define the action class Bookshelf::Actions::Home::Show in app/actions/home/show.rb
+     # ./spec/requests/root_spec.rb:5:in `block (2 levels) in <top (required)>'
+
+Finished in 0.01871 seconds (files took 0.62516 seconds to load)
+1 example, 1 failure
+```
+
+As this error suggests, we need to create the home show action the route is expecting to be able to call.
+
+Hanami provides an action generator we can use to create this action. Running this command will create the home show action:
+
+```shell
+bundle exec hanami generate action home.show
+```
+
+We can find this action in our `app` directory at `app/actions/home/show.rb`:
+
+```ruby
+# app/actions/home/show.rb
+
+# frozen_string_literal: true
+
+module Bookshelf
+  module Actions
+    module Home
+      class Show < Bookshelf::Action
+        def handle(*, response)
+          response.body = self.class.name
+        end
+      end
+    end
+  end
+end
+```
+
+In a Hanami application, every action is an individual class. Actions decide what HTTP response (body, headers and status code) to return for a given request.
+
+Actions define a `#handle` method which accepts a `request` object, representing the incoming request, and a `response` object, representing the outgoing response.
+
+```ruby
+def handle(request, response)
+  # ...
+end
+```
+
+In the generated action above, `*` is used for the `request` argument because the action does not currently use the request.
```

Comment:
The example above doesn't have `*` in the signature

Reply:
Ta. I've made this more clear hopefully.